shell/src/main/java/tachyon/shell/command/MkdirCommand.java

```diff
@@ -66,4 +66,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "mkdir <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Creates a new directory specified by the path in args, including any parent folders that are required.";
```

Comment:
`Creates the specified directory, including any parent directories that are required.`